product-features/self-service-listing.md

```diff
@@ -1,122 +1,176 @@
 
-#### **Step 1.5 Submit**
+</details>
 
-Click to submit Pool information and deposit the initial liquidity of the Anchor Token.
+<details>
 
-*   Example
+<summary>Step 1.2: Confirm token information provided by the contract</summary>
 
-    PEPE deposit 1,800 STX
+Verify that the token information retrieved from the contract is correct. In the example:
 
-### Step 2：Waiting for pool contract creation
+- **Token name** -> `RABBY Token`
+- **Token symbol** -> `RABBY`
+- **Description** -> Unlock the potential of programmable adventures within Bitcoin's rabbit holes.
+- **Token deployment address** -> `SP108J6F4C7JD93BGJ91TEB5D3CFB5XW39QHDJ3MV`
+- **Token logo**
 
-* Once the pool contract is created, 'Contract Ready' status will display 'Ready'.
+</details>
 
-<figure><img src="../.gitbook/assets/image (9).png" alt=""><figcaption></figcaption></figure>
+<details>
 
-### Step3：Deposit Token X
+<summary>Step 1.3: Set the initial liquidity and price</summary>
 
-* After creating the contract, click on 'Deposit' to add an amount of Token X matching the amount you previously submitted.
-  *   Example
+Enter the initial balances for both tokens. You can experiment with different amounts to observe how the exchange rate changes, though we recommend calculating these values beforehand.
 
-      ```
-       PEPE needs to deposit 1,000,000 $PEPE token
-      ```
+In the screenshot example, this is:
 
-<figure><img src="../.gitbook/assets/image (10).png" alt=""><figcaption></figcaption></figure>
+- **Anchor Token ⚓** (a.k.a `token-x`) -> `4,000 STX ($7,200)`
+- **Listing Token 🚀** (a.k.a `token-y`) -> `200,000 RABBY`
+- **Exchange Rate ⚖️** -> `1 RABBY = 0.02 STX ($0.03)`
 
-### Step4：Open Pool
+Once the pool opens, the AMM algorithm will automatically rebalance the exchange rate as users trade the tokens.
 
-Once the 'Open Pool' status appears as "Succeeded," it indicates that the pool has been launched and is listed under ALEX Pool -> self-service pool ([https://app.alexlab.co/pool](https://app.alexlab.co/pool) ). You can now proceed with trading your token on the ALEX DEX.
+</details>
 
-<figure><img src="../.gitbook/assets/image (11).png" alt=""><figcaption></figcaption></figure>
+<details>
 
-### **Step5：Provide additional token information**
+<summary>Step 1.4: Advanced pool settings (optional)</summary>
 
-Click “Contact Us” or "Customer Support" in the Self-Service Listing guide, to add social media information such as Twitter, Official website, etc.
+This step is optional, as the default settings are usually sufficient.
 
-ALEX requires a “Coingecko” or “CoinMarketcap” token listing to verify the provided social media information.
+However, we recomend consulting the [ALEXGo Technical documentation](https://docs.alexgo.io/automated-market-making/trading-pool) before making customizations. If you have any questions, feel free to reach out via [Discord](https://discord.com/invite/alexlab) or [Telegram Channel](https://t.me/AlexCommunity).
```

Comment:
we don't recommend(there is also a typo of recomend in the original wording) users consulting us. 
Just say, If you have questions to ask before customization, reach out via Discord or Telegram